app/main.py

```diff
@@ -1,6 +1,19 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list[dict],
+                 hall_number: int,
+                 cleaner: str,
+                 movie: str
+                 ) -> None:
+    cinema_hall_instance = CinemaHall(hall_number)
```

Comment:
```suggestion
    cinema_hall = CinemaHall(hall_number)
```
here is redundant because you do not have already variable named cinema_hall